Source/Swig/misc.c

```diff
@@ -1477,12 +1483,12 @@ String *Swig_string_regex(String *s) {
 
   DohDelete(pattern);
   DohDelete(subst);
-  pcre_free(compiled_pat);
+  pcre2_code_free(compiled_pat);
   return res ? res : NewStringEmpty();
 }
 
 String *Swig_pcre_version(void) {
-  return NewStringf("PCRE Version: %s", pcre_version());
+  return NewStringf("PCRE Version: %d.%d", PCRE2_MAJOR, PCRE2_MINOR);
```

Comment:
This now reports the PCRE2 version compiled against, whereas before it would report the PCRE version in use at runtime (which may be different to the one compiled against if libpcre2 is a shared library).   The version actually being used is more interesting as bugs are much more likely to be in the library code than inlined by headers.

It looks like the equivalent to `pcre_version()` is something like (untested):

```
    int len = pcre2_config(PCRE2_CONFIG_VERSION, NULL);
    char *buf = malloc(len);
    String *result;
    pcre2_config(PCRE2_CONFIG_VERSION, buf);
    result = NewString(buf);
    free(buf);
    return result;
```